tools/RAiDER/cli/raiderDelay.yaml

```diff
@@ -0,0 +1,60 @@
+# vim: set filetype=yaml:
+##------------------------ raiderDelay.cfg ------------------------##
+########## 1. Date and time
+## Dates can be a single date, two dates that define a set of dates every day, or two dates with an interval
```

Comment:
1) in this comment: the date can also be the date_list 
2) Would be good per category to clarify what is minum to be specified.
So here i am unclear what if i give a date_list then i assume that date_start is not required.

Reply:
@dbekaert good point. I'll add those comments!